Yoast/ruleset.xml

```diff
@@ -108,6 +111,10 @@
 		</properties>
 	</rule>
 
+	<!-- In contrast to WPCS: demand short arrays. -->
+	<rule ref="Generic.Arrays.DisallowLongArraySyntax"/>
+
+
```

Comment:
There is a bonus newline?

Reply:
Sort of. In these config files, I normally have one new line between rules and two new lines before the start of a new section.

The extra new line fixes a minor inconsistency regarding that for this file.

Reply:
I see 👍